### PR TITLE
CHANGELOG-1.17: add note about service CIDR bug

### PR DESCRIPTION
**Pull request by neolit123**
**What this PR does / why we need it**:

Adding note to the changelog to notify users about the problem.

original issue:
https://github.com/kubernetes/kubernetes/issues/86497
fix PR:
https://github.com/kubernetes/kubernetes/pull/86534
a suspect new issue caused by the same bug:
https://github.com/kubernetes/kubernetes/issues/86754

**Which issue(s) this PR fixes**:

NONE

**Special notes for your reviewer**:
NONE

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

/sig release network
/kind documentation
/assign @liggitt 
@kubernetes/sig-release

### CODE REVIEW

File: CHANGELOG-1.17.md [hunk 1]
```diff
@@ -152,7 +153,7 @@ Prior to 1.17 and for existing nodes created by `kubeadm init` where kubelet cli
 - All nodes need to be drained before upgrading Kubernetes cluster, because paths used for block volumes are changed in this release, so on-line upgrade of nodes aren't allowed. ([#74026](https://github.com/kubernetes/kubernetes/pull/74026), [@mkimuram](https://github.com/mkimuram))
 
 #### Windows
-- The Windows containers RunAsUsername feature is now beta. 
+- The Windows containers RunAsUsername feature is now beta.
```
Comment (neolit123) [inline CHANGELOG-1.17.md]: includes auto-whitespace cleanup from my editor.

**Comment by sftim** [comment]
/sig docs
If this merges, please [open an issue](https://github.com/kubernetes/website/issues/new?labels%5B%5D=kind%2Fbug&template=bug-report.md) against the website to track getting that updated as well.

**Comment by neolit123** [comment]
@sftim 
> If this merges, please open an issue against the website to track getting that updated as well.

do you mean documenting that 1.17.0 has the issue and it will be fixed in 1.17.1 in related pages such as https://kubernetes.io/docs/concepts/services-networking/service/#choosing-your-own-ip-address

i don't believe we have done this for PATCH level fixes before.

**Comment by saschagrunert** [comment]
We will have to fix this for the kubernetes.io as well as the relnotes.k8s.io Version. No worries, I can take care of this (already on my todo list).

**Comment by alejandrox1** [comment]
/test pull-kubernetes-e2e-gce
prev failure https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/86759/pull-kubernetes-e2e-gce/1212851802802753537/

**Comment by neolit123** [comment]
`pull-kubernetes-e2e-gce` seems to be failing every time in a different test:
https://prow.k8s.io/pr-history/?org=kubernetes&repo=kubernetes&pr=86759

> [sig-storage] In-tree Volumes [Driver: local][LocalVolumeType: dir-bindmounted] [Testpattern: Pre-provisioned PV (default fs)] subPath should support existing directories when readOnly specified in the volumeSource 

> [sig-network] Services should have session affinity work for service with type clusterIP [LinuxOnly]

> [sig-storage] In-tree Volumes [Driver: local][LocalVolumeType: blockfs] [Testpattern: Pre-provisioned PV (default fs)] volumes should allow exec of files on the volume expand_more	1m37s
> [k8s.io] Pods should support pod readiness gates [NodeFeature:PodReadinessGate] expand_mor

> [sig-api-machinery] ResourceQuota should create a ResourceQuota and capture the life of a replication controller. [Conformance] 

> [sig-storage] Projected configMap should be consumable from pods in volume as non-root [NodeConformance] [Conformance] expand_more	49s
> [sig-storage] Downward API volume should provide container's cpu limit [NodeConformance] [Conformance] expand_more	43s
> [sig-api-machinery] CustomResourcePublishOpenAPI [Privileged:ClusterAdmin] works for multiple CRDs of different groups [Conformance] expand_more

> [k8s.io] [sig-node] Pods Extended [k8s.io] Delete Grace Period should be submitted and removed [Conformance] expand_more	1m1s
> [sig-storage] In-tree Volumes [Driver: nfs] [Testpattern: Dynamic PV (default fs)] subPath should support readOnly directory specified in the volumeMount expand_more	1m27s

**Pull request merged**